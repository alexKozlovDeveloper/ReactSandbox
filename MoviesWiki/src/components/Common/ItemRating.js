import React, { Component } from "react";

import styles from "../../styles/ItemRating.css"

function ItemRating(props) {
    return  <div className={styles.title}>
                {props.rating}
            </div>;
}

export default ItemRating;