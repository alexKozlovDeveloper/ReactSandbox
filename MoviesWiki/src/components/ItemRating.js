import React, { Component } from "react";

//import styles from "../styles/ItemImage.css"

class ItemRating extends Component {
    render() {
        
        //className={styles.imagecontainer}
        
        return (
            <div>
                {this.props.rating}
            </div>
        );
    }
}

export default ItemRating;