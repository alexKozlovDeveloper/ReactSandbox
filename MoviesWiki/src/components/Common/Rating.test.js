import React from 'react';
import { shallow, configure } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import ItemRating from "./ItemRating";

configure({adapter: new Adapter()});

test('ItemRating rendering test', () => {

    var rating = "someRating";

    const component = shallow(<ItemRating rating={rating} />);
    expect(shallowToJson(component)).toMatchSnapshot();
 });

