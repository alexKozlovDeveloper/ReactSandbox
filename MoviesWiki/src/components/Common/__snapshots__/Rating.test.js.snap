// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rating rendering test 1`] = `
<div
  className="title"
>
  someRating
</div>
`;
